package.json

```diff
@@ -10,7 +10,8 @@
     "build": "yarn prep && yarn cp:assets && tsc",
     "prep": "rm -rf build && mkdir build",
     "dev": "yarn build --watch",
-    "cp:assets": "cp -r icons manifest.json build/"
+    "cp:assets": "cp -r icons build/ && yarn substenv:manifest",
+    "substenv:manifest": "sh -ac '. .env; perl -p -e '\"'\"'s/\\$\\{([^}]+)\\}/defined $ENV{$1} ? $ENV{$1} : $&/eg; s/\\$\\{([^}]+)\\}//eg'\"'\"' manifest.json' > build/manifest.json"
```

Comment:
I think this is part of build process instead of copying assets. What are your thoughts?

Reply:
Maybe [this](https://www.npmjs.com/package/envsub) can simplify the scripts. I am trying to avoid complex scripts since they are error prone.

Reply:
Yeah, that's right. It should be part of build. And yeah, envsub package looks cool. I shall change it.